hrs-portlets-ps-impl/src/main/resources/app-context/psAppContext.xml

```diff
@@ -64,6 +64,24 @@
           </set>
         </entry>
 
+        <entry key="UW_DYN_BN_EBENEFITS">
+          <set>
+            <value>ROLE_VIEW_BENEFITS</value>
```

Comment:
Maybe worth iterating on this role name a bit. Maybe parallel to the below it could be `ROLE_VIEW_ADHOC_ENROLL_BENEFITS`? Or we embrace the PeopleSoft terminology and get "EBENEFITS" in there somewhere. `ROLE_VIEW_BENEFITS` feels too much like it might be the role of being able to view Benefit Information at all or so, but that's not what this role is.